Flashcards.Deviationlark/App.config

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <appSettings>
+        <add key="connectionString" value="Data Source = (LocalDb)\LocalDBDemo; initial catalog = Flashcards; integrated security = true;"/>
+    </appSettings>
+
+    <connectionStrings>
+        <add name="connString" connectionString="Data Source = (LocalDb)\LocalDBDemo; initial catalog = Flashcards; integrated security = true;"/>
```

Comment:
I see the `connectionString` is duplicated. In your code you use `connString`, I actually prefer the verbose `connectionString`, it is easier to read and avoids confusion to use very verbose names.